common/changes/iconame-reducesize_2017-01-10-01-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Icon: -BREAKING CHANGE- Change IconName from an enum to type to greatly reduce file size. Any component that is currently consuming the icon enum will break.",
```

Comment:
I'm probably wrong here. If others are ok, let's major bump the whole package. I think it will be helpful for consumers to be really conscious when we make breaking change.s